rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -3099,6 +3099,18 @@ namespace rsx
 			Emu.Pause();
 		}
 
+		if (false && wait_for_flip_sema) // Breaks framepacing
```

Comment:
With the way GCM seems to work, submitting multiple frames in one vblank window does not advance the timeline semaphore twice, making this block useless. I was hoping each head would get its own semaphore but of course nothing is ever easy with RSX.